versions/3.1.0.md

```diff
@@ -3162,25 +3163,109 @@ animals:
 </aliens>
 ```
 
+#### <a name="alternativeSchemaObject"></a>Alternative Schema Object
+
+This object makes it possible to reference an external file that contains a schema that does not follow the OAS specification. If tooling does not support the ``type``, tooling MUST consider the content valid but SHOULD provide a warning that the alternative schema was not processed.
+
+##### Fixed Fields
+Field Name | Type | Description
+---|:---:|---
+<a name="alternativeSchemaType"></a>type | `string` | **REQUIRED**. The value MUST match one of the values identified in the alternative Schema Registry.
+<a name="alternativeSchemaLocation"></a>location | `url` | **REQUIRED**.  This is a absolute or relative reference to an external resource containing a schema of a known type.  This reference may contain a fragment identifier to reference only a subset of an external document.
+
+This object MAY be extended with [Specification Extensions](#specificationExtensions).
+
+##### Examples
+
+Minimalist usage of alternative schema:
+
+```yaml
+schema:
+  alternativeSchema:
+    type: jsonSchema
+    location: ./real-jsonschema.json
+```
+
+Combination of OAS schema and alternative:
+
+```yaml
+schema:
+  type: object
+  nullable: true
+  alternativeSchema:
+    type: jsonSchema
+    location: ./real-jsonschema.json
+```
+
+Multiple different versions of alternative schema:
+
+```yaml
+schema:
+  anyOf:
+  - alternativeSchema:
+      type: jsonSchema
+      location: ./real-jsonschema-08.json
+  - alternativeSchema:
+      type: jsonSchema
+      location: ./real-jsonschema-07.json
+```
+
+Combined alternative schemas:
+
+```yaml
+schema:
+  allOf:
+  - alternativeSchema:
+      type: xmlSchema
+      location: ./xmlSchema.xsd
```

Comment:
I'd add a `fragment` to better clarify https://github.com/OAI/OpenAPI-Specification/pull/1736/files#diff-b83206d953b4be5d312b068f82ba0217R3174

eg. see https://github.com/teamdigitale/api-openapi-samples/blob/alternativeSchema/openapi-v3/external-schema.yaml

```
            anyOf:
            - alternativeSchema:
                type: xmlSchema
                location: schema.xsd#/person
```
@andreapoli